rusk-recovery/src/state/snapshot/stake.rs

```diff
@@ -14,7 +14,8 @@ use super::wrapper::Wrapper;
 
 #[derive(Serialize, Deserialize, PartialEq, Eq)]
 pub struct GenesisStake {
-    pub(crate) address: Wrapper<BlsPublicKey, { BlsPublicKey::SIZE }>,
+    address: Wrapper<BlsPublicKey, { BlsPublicKey::SIZE }>,
+    owner: Option<Wrapper<BlsPublicKey, { BlsPublicKey::SIZE }>>,
```

Comment:
What is the difference between an address and owner? Is the owner the unlocker of stakes belonging to the given address?

Reply:
The address is the stake address

This is just to support stake keys segregation on genesis

Reply:
> The address is the stake address (I didn't change the name to be backward compatible)
> 
> This is just to support stake keys segregation on genesis

Nevermind! Looked up the `StakeKeys::new` signature and now I get the bigger picture again. Thanks!